src/content/pages/ecosystem-explorer/project-form.md

```diff
@@ -3,8 +3,8 @@ header:
   title: "Add a Project"
   description:
     [
-      "Let's get started with some basic information about what you're building. Please submit or update your entry to the Filecoin ecosystem page below.",
-      "Want to be a featured project? Share your project page on X and tag @filfoundation to be considered!",
+      "Let's get started with some basic information about what you're building. Please submit your project to the Filecoin ecosystem page below. To update an existing project, send an email to info@fil.org.",
```

Comment:
Should go into siteMetadata.ts - Should also be a mailto link.

Reply:
@mirhamasala Do the new changes work?